src/clang-c-frontend/clang_c_main.cpp

```diff
@@ -70,17 +70,29 @@ bool clang_c_maint::clang_main()
     // if user provided class/contract name
     if (!config.cname.empty() && !config.main.empty())
     {
-      const std::string fmt = "@" + config.cname + "@F@" + main;
-      if (it->second.as_string().find(fmt) == std::string::npos)
+      // Allow user to provide a range. e.g.
```

Comment:
Can you provide a test case in C++ that checks this new "feature"?

Reply:
Added as suggested